pom.xml

```diff
@@ -107,6 +110,9 @@
             <goals>
               <goal>jar</goal>
             </goals>
+            <configuration> <!-- There are some malformed javadoc comments -->
```

Comment:
Non-blocking: could you specify where? I didn't run into any issues when I last built this.